src/Components/ProductTile.js

```diff
@@ -1,6 +1,15 @@
+import React, { useState } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8496218)
```suggestion
import React, { useState } from "react";
```